var/spack/repos/builtin/packages/eem/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Eem(MakefilePackage):
+    """EEM is a program to extract the gene group and expression modules
+       that are co-expressed in the mRNA expression data."""
+
+    homepage = "http://www.scls.riken.jp/scruise/software/eem.html"
+    url      = "http://www.scls.riken.jp/wp-content/uploads/2014/07/eemParallel_1.0.1.tar.gz"
+
+    version('1.0.1', sha256='f617ea7350fce3b2581c814f70bda4427cbab83aac54a2dcadb36e8193f300bb')
+
+    variant('K', default=False, description='Build for K computer')
+
+    phases = ['edit', 'install']
```

Comment:
What happens when you run `make` with no arguments?

Reply:
`make` with no argument executes the build twice.
However, the installation can be performed without problems.
Is `phases` unnecessary?

Reply:
Hmm, building twice sounds like a waste of time. I'm fine with your use of `phases` then.